test/e2e/tests/phishing-detection.spec.js

```diff
@@ -127,4 +138,30 @@ describe('Phishing Detection', function () {
       },
     );
   });
+
+  it('should display the MetaMask Phishing Detection page with the correct new issue link if the issue was detected from the phishfort list', async function () {
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+        testSpecificMock: mockPhishfortPhishingDetection,
+        dapp: true,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        await driver.openNewPage('http://127.0.0.1:8080');
+        const newIssueLink = await driver.findElement({
+          text: 'please file an issue',
+        });
+        assert.equal(
+          await newIssueLink.getAttribute('href'),
```

Comment:
The `href` value here might be the new GitHub ticket link that will be created to report the issue.